_docs/ops/runbook/restoring-rds.md

```diff
@@ -10,35 +10,62 @@ title: Restoring RDS
 
 ## Customer Databases
+
 Coordinate with the tenant to determine the point in time to restore the database from, and when the operation will take place:
 
 1. Shared database plans do not include backups and restores.
 1. Dedicated RDS plans include backups as described in the [user documentation]({{ site.baseurl }}{% link _docs/services/relational-database.md %}#backups).
 1. The restore will result in the database being restored to a point in time.
 1. Data written after the restore point in time will be lost.
 
+### Obtain information and confirmation from the customer
+
+In order to perform a restore, we need the following information from the customer:
+
+- The organization name
+- The space they are working within
+- The name of the application(s) connected to the database service they need a restoration performed on
+
+Be sure to confirm this information and remind them that a restoration may result in a brief period of downtime with database connectivity.  Ask if they have some way of shutting off access/enabling a maintenance mode of the website and if so, they should do so prior to the restoration process starting.
+
+If the customer agrees to proceed given this information, coordinate with them on the date and time to perform the restore and with which PITR (point in time restore) or snapshot to use.  At the agreed upon date and time, continue with the steps below.
+
 ### Identify RDS Hostname
-Once the tenant agreed to a database restore, identify the RDS instance attached to the application:
+
+Once the tenant agreed to a database restore, identify the RDS instance attached to the application using the information they provided:
+
 ```sh
 cf target -s SPACE -o ORGANIZATION
 cf env APP
 ```
 
-The JSON contains the **database identifier** under the key `host` within the credentials map.
+The JSON containing the environment variables contains the `database identifier` and `password` under the key `host` within the credentials map for the database service (look for `aws-rds`).  The `database identifier` is needed to find the database instance in the AWS RDS instance listing, and the `password` is required for setting the master password in the new instance.
+
+### Performing the database restore
 
-## Restoring the Database
 Refer to the [RDS documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_RestoreFromSnapshot.html) for restoring a database.
 
-Prior to restoring, record the configuration settings:
-1. Database publicly accessible from the internet (yes/no)
-1. Instance size (m4.large, etc.)
-1. Multi-zone (yes/no)
-1. VPC (dev, staging, etc.)
-1. Security Groups
+Prior to restoring, record the configuration settings of the existing instance:
+- Database publicly accessible from the internet (yes/no)
```

Comment:
Let's remove this as none of our DBs are publicly accessible.

Reply:
That's true - I think it's still worth noting to make sure that the DB is not set to publicly accessible, but I removed it from this list and turned it into a note to double check the setting to make sure it's set properly, in case the defaults ever change.